### ISSUE TITLE
Upload bulk users (CSV): preview content displays under side blocks

### ISSUE THREAD

**Issue by nadavkav** [gjb2048/moodle-theme_essential#276]
While uploading some users using a CSV file.
the preview table wrapper div is displaying the table's content under the blocks.
(moodle/admin/tool/uploaduser/index.php)
It should have:
```
#page-admin-tool-uploaduser-index .flexible-wrap {overflow:auto;}
```

**Comment by mkpelletier** [comment]
I realise this is an old one, so perhaps it has been fixed in the core.  I cannot seem to replicate it now in essential 2.7.9.4.3

**Comment by gjb2048** [comment]
Thanks.

**Comment by nadavkav** [comment]
Fixed, testing Essential theme (2.8.1.1) on latest Moodle master (15-1-2015).

Though, I noticed a new issue... No horizontal scroll bar on "preview table" when there are more columns than can be displayed on screen.
